vms/platformvm/blocks/executor/proposal_block_test.go

```diff
@@ -95,6 +95,7 @@ func TestApricotProposalBlockTimeVerification(t *testing.T) {
 		NodeID:    utx.NodeID(),
 		SubnetID:  utx.SubnetID(),
 		StartTime: utx.StartTime(),
+		NextTime:  chainTime,
```

Comment:
in RewardValidatorTx I replaced EndTime with NexTime, so gotta duly initialize these attributes in tests